envs/vagrant/defaults.yml

```diff
@@ -65,13 +65,61 @@ client:
+  client_cert: |
+    -----BEGIN CERTIFICATE-----
+    MIIC3TCCAcWgAwIBAgIBAjANBgkqhkiG9w0BAQUFADASMRAwDgYDVQQDEwdTZW5z
+    dUNBMB4XDTE1MDEwNTIxMzYyM1oXDTIwMDEwNDIxMzYyM1owITEOMAwGA1UEAwwF
+    c2Vuc3UxDzANBgNVBAoMBmNsaWVudDCCASIwDQYJKoZIhvcNAQEBBQADggEPADCC
+    AQoCggEBAPOCFDzom4sBkFbpy7D6OZX2Re/mbbcpdz7zAYR27X8WE7ai4FCe/OlW
+    lViHBkUdHl8TigYBLpZOX+BBu8UMFThotaq10+E07JQygeZLF/Yg6ihoC2qq1n2w
+    Nan2CkhTdKwYfzeMM14Kdc/cu+1TG64kZQQ6j/TCSho9n1BinxbjJi+Gnp34gDLY
+    c3pI/dpxflPNo9PiPD3XS0WUk1gJm1anNOJLdipGEZYW1fTXL3XciLyrEFEV3am1
+    pUFyD7l+ygHtUfPGJFc+i7iuHqPBU/HvSeIRd/5ynLIqJmNHzmtrUiRiie3TVIjo
+    6PHTLI9l/YS20TGuNQTWnYEFZp0WGtUCAwEAAaMvMC0wCQYDVR0TBAIwADALBgNV
+    HQ8EBAMCB4AwEwYDVR0lBAwwCgYIKwYBBQUHAwIwDQYJKoZIhvcNAQEFBQADggEB
+    AGmzryodqCedFin/9ytQerlAf4To3B1j+F/FSoQfnm7smRV3dz7FRr7vlNhX4Spg
+    p2AQAPV+PBobtvSToDZZGj5Puwog/lfX9PIcAI0JoQOi+ylXjMybJtGSC513fVEj
+    4h5isvAtLEtiEB0czVhF5REXwzFIMTSwTDuCAxgfvaZqkn6J7R5NChcQUpsHxeP+
+    TAuDsm+7LJC8K7mAYS4sCi1VR1Wmq7rF3Wqoe2PdzYg9jKzWDhoBrL+cHZ/CifoY
+    GH4uMZWaTZgJuDNCseRjRwrONFcJqdFm0IAFCt/8r8dlgLL8G3Uu2ABJt+lZvvHf
+    NaVOvEPwUrNq9IJtmam/Wlw=
+    -----END CERTIFICATE-----
+  client_key: |
+    -----BEGIN RSA PRIVATE KEY-----
+    MIIEowIBAAKCAQEA84IUPOibiwGQVunLsPo5lfZF7+Zttyl3PvMBhHbtfxYTtqLg
+    UJ786VaVWIcGRR0eXxOKBgEulk5f4EG7xQwVOGi1qrXT4TTslDKB5ksX9iDqKGgL
+    aqrWfbA1qfYKSFN0rBh/N4wzXgp1z9y77VMbriRlBDqP9MJKGj2fUGKfFuMmL4ae
+    nfiAMthzekj92nF+U82j0+I8PddLRZSTWAmbVqc04kt2KkYRlhbV9NcvddyIvKsQ
+    URXdqbWlQXIPuX7KAe1R88YkVz6LuK4eo8FT8e9J4hF3/nKcsiomY0fOa2tSJGKJ
+    7dNUiOjo8dMsj2X9hLbRMa41BNadgQVmnRYa1QIDAQABAoIBAD5Ju9uItKDuGXnh
+    /BSdLTXK9UDUdKQxmysjBfRYZ4/mmcxSBYcZ5B/b4f/vKg9rz2UKnnfCvaP9EFFQ
+    /zL8UEFJp46vTC+DVjDiao/6DXsMGFmVo7X5VTx5YoDDSwYNMv8ClSgkWM9pZZFy
+    KTAf3ijBboWSZVh6inID6UiqsX8sw8ZjHdexT2/b29/Q64nLKTrG472F0mACMCqT
+    TTNUSLCH78blAzySyZaK9miL7ehvxMTv2MfNwspg7vVV4PRtqlM0+bnYsz2aLkQ1
+    gMNzVTuyux1lg0mj+xst9aKpQ4ZGmfRT7MPr9USa65k2vakaik6DaAQNkS+C8l74
+    +JcvyIECgYEA/9ILhig/vvHxch7D8eq9sNXld0NNhZd9PKbb05t8K0zeCXTXVM4d
+    DQ1SLErpae6yFcTGWJWdSua6mClURGm2W10JH+QLUNq51Drl++Kj1scUSADVpjYZ
+    JkmvvRjGlLX5bzJ3mBGbmdKyGdMeppZRWk3+jhGuEMTo0z4URaDllE0CgYEA863S
+    gJBd7/bhVCFbNuXEATMjoXjLsKK9QPDjBaL/PEnKoYT1aTsVUN43Hkd86c5NXszR
+    ///IzA49TluKHvXjscLk2mzInUU3YyjwzWgN5vp6ZEh0bubXWABgQVVFsmqV3Gzp
+    KBrBg9mzWtC6KQGwtt3RRInFycTYE6d+uf8OBKkCgYEAv8PNXbC4MEOfMWO8kJDd
+    xzxf+ana58SqXZfa+2H3j2Hco4wRioHDJzCQI6G4HO+QUV8jK+5jbW1N/Jgmke4I
+    g65XE6/Xh7GeUWghVFMNbAfpvRvbC/BLo/bipMZ4vas3otJa8gRo9sMJPPCUbl4J
+    9761jfvU1r80pH1JpvQ4hJUCgYAS+zpHe3dlxtRcilChoRo4gbRH0rIDK1oUoe79
+    NGovVwwqssGvFcQeTYD7odPwHnrWZJwDFfidNIq//M9wg/TdlvHetdSWs5qR5dGE
+    HpPepyo7f41aSi8CEt1smWjZcgYEapNq5VKno6Cd9V5a7V/HjtXLQfQfOG8gcM6l
+    TGHaKQKBgFRnRUE3tBeEKvM1touGxNjURea4aFVzFk4b8lL/OQJnkWhMmhrBKCHw
+    WRiRItwHQxTunlN0gK4qty2rczWIv/yzWALHJAOFLIyX0ynO4xY7x/3H9df9IuDe
+    Gs1548GcEmUG0CWujFXsEJkjjBxT4/fabx4NOJjg9Ij1Mi3LYmB2
+    -----END RSA PRIVATE KEY-----
```

Comment:
nope, this is the cert set that I generated for the rabbit server in ursula-infra